tests/optimize/test_spsa.py

```diff
@@ -479,7 +479,7 @@ def cost_fun(params, num_qubits=1):
         assert energy < init_energy
 
     @pytest.mark.slow
-    def test_lighting_device(self):
+    def test_lightning_device(self):
```

Comment:
In this test, a kwarg is being used but the bug was not caught. May I ask you to add an assertion that would fail on master? :)

Reply:
@dwierichs how's that?

Reply:
Thanks!